src/pages/VerifyPhone/CodeForm/CodeForm.jsx

```diff
@@ -0,0 +1,94 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import React, { useState } from 'react';
+import { CountdownCircleTimer } from 'react-countdown-circle-timer';
+import { useForm } from 'react-hook-form';
+import colors from 'styles/colors';
+import {
+  ButtonsContainer,
+  CancelButton,
+  CodeContainer,
+  DigitInput,
+  FormContainer,
+  InputsContainer,
+  ResendButton,
+  ResendCircleContainer,
+  TimerText,
+  VerifyButton,
+} from './code-form.styles';
+import codeFormSchema from './CodeFormSchema';
+
+function CodeForm() {
+  const [resendCode, setResendCode] = useState(true);
+
+  const { register, handleSubmit, reset } = useForm({
+    resolver: yupResolver(codeFormSchema),
+  });
+
+  const progressBarTimer = ({ remainingTime }) => {
+    const minutes = Math.floor(remainingTime / 60);
+    const seconds = remainingTime % 60;
+    return (
+      <TimerText>
+        {minutes}:{seconds}
+      </TimerText>
+    );
+  };
```

Comment:
Please remove pure functions like this one outside of the component's function scope.
We should also restraint from using names for such functions and use verbs instead. 
```suggestion
  const renderProgressBar = ({ remainingTime }) => {
    const minutes = Math.floor(remainingTime / 60);
    const seconds = remainingTime % 60;
    return (
      <TimerText>
        {minutes}:{seconds}
      </TimerText>
    );
  };
```